.github/workflows/fih_tests.yaml

```diff
@@ -13,13 +13,17 @@ concurrency:
 jobs:
   config:
     strategy:
+      fail-fast: false
```

Comment:
Why?

Reply:
This reduces the amount of times you need to run the whole CI, if there are different errors between different configs, but @tamasban can clarify it more.

Reply:
I have never used this option, but I think that failing fast is better than failing slow! :rofl:

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

Maybe what you want to achieve is to let other workflows run to the end even if the FIH tests fail? @tamasban

Reply:
AFAIU this config option has no effect on other workflows like sim, Zephyr, etc. Its scope is restricted only to the FIH jobs.

The goal is to let the other jobs run until the end within the FIH test suite even though one fails. With this approach, one can see all build or FI-related issues after a single CI execution and failures are not revealed only one by one in consecutive CI executions.

This is not aligned with the rest of the CI job configs. We can revert this change if you want to keep all the jobs config aligned.